public/js/clients/chrome.js

```diff
@@ -184,9 +192,24 @@ function initPage(actions) {
   runtimeAgent.run();
 }
 
+function injectAgent() {
+  if (isAgentInjected) {
+    return;
+  }
+  const script = `window.Agent = (${Agent})()`;
+  console.log(script);
+  runtimeAgent.evaluate(script, (_, result) => {
+    isAgentInjected = true;
+    console.log(result);
+  });
+}
+
+window.injectAgent = injectAgent;
```

Comment:
Yeah, we could export a getter for `runtimeAgent` if you want to do this from somewhere else